controllers/articles.controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchArticleById } = require("../models/articles.model");
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+
+  fetchArticleById(article_id)
+    .then((article) => {
+      res.status(200).send(article[0]);
```

Comment:
You should send back article in an object here, also if you return rows[0] in your model then you don't have to use article[0] here, you should just have article